src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.Azure.Management.Analysis.Models
     /// </summary>
     public static class SkuName
     {
+        public const string B1 = "B1";
+        public const string B2 = "B2";
```

Comment:
I would really prefer not to do it now. There is just one scenario test which tests multiple actions one by one (provision->update->delete), it does not really test different skus, it just provisions a server with first sku. We have already many sku constants defined and I just added extra more. In the near future we want to switch to dynamic skus loading when they will be loaded from REST call, then we will definitely implement more advanced sku testing.